mllib/src/main/scala/org/apache/spark/ml/recommendation/ALS.scala

```diff
@@ -656,13 +656,15 @@ object ALS extends DefaultParamsReadable[ALS] with Logging {
         previousItemFactors.unpersist()
         itemFactors.setName(s"itemFactors-$iter").persist(intermediateRDDStorageLevel)
         // TODO: Generalize PeriodicGraphCheckpointer and use it here.
+        val deps = itemFactors.dependencies
```

Comment:
really minor, but let's move this a line down under `if (shouldCheckpoint(iter)) { `, just to match the explicit case below